scripts/zowe-api-mediation-configure.sh

```diff
@@ -155,21 +170,69 @@ services:
           serviceRelativeUrl: api/v1/jobs
         - gatewayUrl: ui/v1  # [api/ui/ws]/v{majorVersion}
           serviceRelativeUrl: ui/v1/jobs
+      apiInfo:
+        - apiId: com.ibm.jobs
+          gatewayUrl: api/v1
+          version: 0.9.3
+          documentationUrl: https://$ZOWE_EXPLORER_HOST:$ZOWE_EXPLORER_SERVER_HTTPS_PORT/ibm/api/explorer/
+catalogUiTiles:
+    jobs:
+        title: z/OS Jobs services
+        description: IBM z/OS Jobs REST services
 EOF
 iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/jobs.yml > $STATIC_DEF_CONFIG/jobs.yml	
 
+# Add static definition for uss
+cat <<EOF >$TEMP_DIR/uss.yml
+#
+services:
+    - serviceId: uss
+      title: IBM z/OS USS
```

Comment:
Can you change the title/description to z/OS Unix Files as I think that'll be clearer to people

Reply:
Makes sense